### DISCUSSION TITLE
quick/install

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#23] [General]
# quick/install

大家好，欢迎访问 GoFrame 框架的快速开始章节！

https://goframe.org/quick/install

**Comment by lkidc** [comment]
学习Goframe第一天

**Comment by demozx** [comment]
新官网，新起点，加油goframe

**Comment by demozx** [comment]
当时学习goframe的项目分享

代码仓库：
https://github.com/demozx/gf_cms
https://gitee.com/demozx/gf_cms

前台演示地址：[demo-pc.linklink.fun](http://demo-pc.linklink.fun/) （自动跳转移动端）
移动演示地址：[demo-m.linklink.fun](http://demo-m.linklink.fun/)
后台演示地址：[demo-pc.linklink.fun/admin](http://demo-pc.linklink.fun/admin)
用户名：admin
密码：123456

**Reply by shayuevil** [reply]
好人一生平安

**Comment by shayuevil** [comment]
新官网看着好舒服啊 冲冲冲

**Comment by zcool321** [comment]
新官网支持一下

**Reply by gqcn** [reply]
狐狸大佬，欢迎多多参与社区建设哦❤️

**Comment by chaggle** [comment]
准备重学goframe

**Comment by ideaviewes** [comment]
有没有可能实现类似于wordpress那样的插件功能呢？

**Comment by ThePeppy** [comment]
加油goframe,祝越来越好
